README.md

```diff
@@ -111,7 +111,7 @@ To organize and steer the development of [helm-operator-plugins](https://github.
 
 ### Meetings
 
-* Issue Triage and Community Meeting: [Tuesdays at 11:00 ET (Eastern Time)](https://github.com/operator-framework/helm-operator-plugins/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=ET%20%28Estern%20Time%29).
+* Issue Triage and Community Meeting: [Tuesdays at 11:00 ET (Eastern Time)](https://github.com/operator-framework/helm-operator-plugins/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=ET%20%28Eastern%20Time%29).
```

Comment:
```suggestion
* Issue Triage and Community Meeting: [Tuesdays at 11:00 AM ET (Eastern Time)](https://github.com/operator-framework/helm-operator-plugins/issues?q=is%3Aopen+is%3Aissue+no%3Amilestone+sort%3Acreated-asc) (biweekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=11:00&tz=ET%20%28Eastern%20Time%29).
```

Reply:
Ignore, looks like we're using 24 hour format. https://github.com/operator-framework/community/blob/master/README.md#meetings-1 so 12:00 ET would indicate noon.